### ISSUE TITLE
Wikipedia language guessing is not very accurate it seems

### ISSUE THREAD

**Issue by RURon** [emvaized/selecton-extension#68]
Hello there!

When I try to look up using the dictionairy function, even though I'm on a german website with a german word, I only get routed to the english Wikipedia site. Maybe you could add an override / site text input just like for custom buttons (of course this would currently also work, but why add another Wikipedia custom button if there already is a nice dictionairy link)

Another thing would be to add an option to "always add map link" as for example when you want to look up a city, you need a custom button for that anyways as there are no keywords to be found (at least not for most city names).

Besides that I really like this extension! Greetings!

**Comment by emvaized** [comment]
Hi there! 

> I only get routed to the english Wikipedia site

Hmm, I believe it used to redirect to the appropriate language page back when I developed this button... Maybe it was changed in the meantime, or it only used to work this way for me. 

Probably it would work to just prepend proper locale in the url? So that, for example, in your case it would be `de.wikipedia.org` instead of just `wikipedia.org`? 

> add an option to "always add map link" 

I do not like this idea, as it will increase button clutter, while being useful in quite a specific use case. And even in this case, "Search" button could usually be used to see the location of the city, with just a few more clicks to see the map (which doesn't work this way all the time with specific addresses, for which the "Map" button is intended for).
